snmp/datadog_checks/snmp/data/default_profiles/huawei-routers.yaml

```diff
@@ -160,9 +161,10 @@ metrics:
           OID: 1.3.6.1.4.1.2011.5.25.194.1.4.1.2
           name: huawei.hwServerStatus
         tag: huawei_hw_server_status
-        mapping:
-          1: static
-          2: dynamic
+        # mapping:
+        #   1: static
+        #   2: dynamic
+        # mapping absent in MIB
```

Comment:
Similar to a previous comment. If the mapping is missing, it's likely better to just comment the whole metric for now.